SDNExpress/scripts/TenantConfig.psd1

```diff
@@ -29,8 +29,9 @@
 
             #Password to assign to the local administrator of created VMs
             VMLocalAdminPassword = '<< Replace >>'                              #Example: "V3ryC0mplexP4ssword"
+            
```

Comment:
The comment should probably go above the UseIdns = $true line, as it's explaining what the feature is, not providing an example.